Sources/SpeziFHIRInterpretation/FHIRResourceSummaryView.swift

```diff
@@ -16,13 +16,14 @@ public struct FHIRResourceSummaryView: View {
     @Environment(FHIRResourceSummary.self) private var fhirResourceSummary
     
     @State private var viewState: ViewState = .idle
+    @State private var cachedSummary: FHIRResourceSummary.Summary?
     
     private let resource: FHIRResource
     
     
     public var body: some View {
         Group {
-            if let summary = fhirResourceSummary.cachedSummary(forResource: resource) {
+            if let summary = cachedSummary {
```

Comment:
What is the main reason not to use the cache right in the summary class? This could be persisted across application launches while the state would be lost across application launches.

Reply:
Good catch, I actually don't remember why I made the change (it doesn't make any sense at all). I think I probably drafted that as some kind of intermediary step and didn't refactor it.